gocd/bci_repo_publish.py

```diff
@@ -190,7 +219,8 @@ def setup_tool(self):
 
         return tool
 
-    @cmdln.option('--token', help='The token for publishing. Does a dry run if not given.')
+    @cmdln.option('--publish-token', help='The token for publishing. Does a dry run if not given.')
+    @cmdln.option('--rebuild-token', help='The token for rebuilding devel:BCI. Does a dry run if not given.')
```

Comment:
The "Does a dry run if not given" is kinda wrong.. maybe just omit it or say something like "trigger a rebuild with this token if given"?

Reply:
I've adjusted the description.